extensions/devservices/mssql/src/main/java/io/quarkus/devservices/mssql/deployment/MSSQLDevServicesProcessor.java

```diff
@@ -35,7 +37,7 @@ public RunningDevServicesDatasource startDatabase(Optional<String> username, Opt
                 QuarkusMSSQLServerContainer container = new QuarkusMSSQLServerContainer(imageName, fixedExposedPort,
                         !devServicesSharedNetworkBuildItem.isEmpty());
                 startupTimeout.ifPresent(container::withStartupTimeout);
-                container.withPassword(password.orElse("Quarkuspassword1"))
+                container.withPassword(password.orElse(DEFAULT_DATABASE_PASSWORD))
```

Comment:
We don't define a username, here?

Reply:
I was also surprised, but did not dig deeper into this one.